### PR TITLE
Page Structure

### PR DESCRIPTION
**Pull request by er-contreras**
- Create an npm project with Webpack.
- Write plain HTML markup with minimum styling

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+import './style.css';
```
Comment (henatan99) [inline src/index.js:1]: - [x] You should construct your DOM elements here. But it looks your index.js is empty. Please make the required changes here.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "build": "webpack"
+  },
```
Comment (henatan99) [inline package.json]: - [x] Your package.json scripts miss the `start` key. You app should run with `npm start` or `npm run start` command.

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @eri8-9 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

- [x] You did not configure the GitHub workflows linters file so linter checks are not showing in your github repo. Kindly fix this.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    name: 'Name',
+    score: '77',
+  },
+  {
+    name: 'Name',
+    score: '42',
+  },
+];
+
+function htmlContent() {
```
Comment (Davidosky007) [inline src/index.js]: * In this project you are encouraged to use ES6 syntax (ie arrow function) [Refer to Project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md#:~:text=using%20webpack%20and%20ES6%20features%2C) i suggest you replace the `function` keyword with `arrow function` as it is a requirement for this project

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
# Hi @eri8-9 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @eri8-9  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Well done team

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**